README.md

```diff
@@ -25,7 +25,7 @@ For more information on role assumption, see the [AWS documentation](https://doc
 
 Requirements
 ------------
-*   Ruby ≥ 2.2
+*   Ruby ≥ 2.3.5
```

Comment:
We still expect `aws-assume-role` to work on Ruby 2.2 and have Travis CI tests for it.
2.3.5 is the default build environment.